### PR TITLE
Add Honeycomb.io Go and Node Distros to Registry

### PR DESCRIPTION
**Pull request by vreynolds**
Add Honeycomb.io OpenTelemetry Go and Node.js Distros to Registry

### CODE REVIEW

File: data/registry/instrumentation-go-honeycomb.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title: Honeycomb.io OpenTelemetry Distro for Go
```
Comment (chalin) [inline data/registry/instrumentation-go-honeycomb.yml:1]: I suggest matching the title at https://github.com/honeycombio/honeycomb-opentelemetry-go#readme:

> # Honeycomb OpenTelemetry Distro for Go

```suggestion
title: Honeycomb OpenTelemetry Distro for Go
```

File: data/registry/instrumentation-js-node-honeycomb.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+title: Honeycomb.io OpenTelemetry Distro for Node.js
```
Comment (chalin) [inline data/registry/instrumentation-js-node-honeycomb.yml:1]: Ditto:

```suggestion
title: Honeycomb OpenTelemetry Distro for Node.js
```

File: data/registry/instrumentation-go-honeycomb.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+registryType: instrumentation
+isThirdParty: true
+language: go
+tags:
+  - go
+  - instrumentation
+repo: https://github.com/honeycombio/honeycomb-opentelemetry-go
+license: Apache 2.0
+description: Honeycomb's distribution of OpenTelemetry for Go
+authors: Hound Technology Inc
```
Comment (chalin) [inline data/registry/instrumentation-go-honeycomb.yml:11]: Should this be the following (which is what is in the NOTICE files of those repos)?

```suggestion
authors: Honeycomb, Hound Technology, Inc
```

File: data/registry/instrumentation-js-node-honeycomb.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+isThirdParty: true
+language: js
+tags:
+  - js
+  - Node.js
+  - instrumentation
+repo: https://github.com/honeycombio/honeycomb-opentelemetry-node
+license: Apache 2.0
+description: Honeycomb's distribution of OpenTelemetry for Node.js
+authors: Hound Technology Inc
```
Comment (chalin) [inline data/registry/instrumentation-js-node-honeycomb.yml:12]: Should this be the following (which is what is in the NOTICE files of those repos)?

```suggestion
authors: Honeycomb, Hound Technology, Inc
```

**Review by chalin — COMMENTED** [review]
Thank you for having resubmitted the PR! ✨ 

See inline for minor comments / questions.

**Comment by chalin** [comment]
If possible, please also rebase so that you can pickup the latest fixes to the registry.

File: data/registry/instrumentation-go-honeycomb.yml (hunk 3 above)
Comment (vreynolds) [reply data/registry/instrumentation-go-honeycomb.yml:11]: Ah, we can include both legal and DBA names 👍

**Comment by vreynolds** [comment]
Rebased and  committed suggestions 🙏

**Comment by chalin** [comment]
@vreynolds - Just curious... You accepted the two suggestions to drop the ".io" from the titles, but then added them both back via your last commit. Was that intentional?

**Review by chalin — APPROVED** [review]
LGTM. Will merge once you confirm whether or not you want the ".io" in the title.

**Comment by chalin** [comment]
Ok, let's get this in as is. Changes can be made later if needed. Thanks again for the PR!

**Comment by vreynolds** [comment]
Apologies, I was unavailable last week. The commit suggestion UI led me astray, I meant to accept you proposal, but now that I look at all the Honeycomb files, I want to make them consistent. I will do a follow-up PR.

**Pull request merged**